website/docs/r/iam_user.html.markdown

```diff
@@ -53,6 +56,7 @@ The following arguments are supported:
 * `force_destroy` - (Optional, default false) When destroying this user, destroy even if it
   has non-Terraform-managed IAM access keys, login profile or MFA devices. Without `force_destroy`
   a user with non-Terraform-managed access keys and login profile will fail to be destroyed.
+* `tags` - Tags for the IAM user
```

Comment:
Nit: This should probably specify key-value pairs or map as a hint to the format of the argument, e.g.

```suggestion
* `tags` - Key-value mapping of tags for the IAM user
```